### DISCUSSION TITLE
Log File Reader application with filtering and color highlighting?

### DISCUSSION THREAD

**Discussion by jjeff** [megahertz/electron-log#386] [Q&A]
I'm on the Mac and it's relatively easy to open electon-log's `main.log` and `renderer.log` files using Apple's "Console" app and watch them tail as my app runs. However, these files can be quite verbose. There's no color highlighting. And there's no way to filter to see (for example) just the `[info]` and lower messages or just the `[error]` messages.

Are there any apps out there that offer this type of functionality?

Or something that might add this type of functionality to the VS Code debug console?

**Comment by megahertz** [comment]
Color highlighting works in Kate. I think it's possible to find some program that hides some lines by RegExp, but I don't know any

**Reply by jjeff** [reply]
https://kate-editor.org/
